src/tests/components/Navbar.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navbar from '../../components/UI/Navbar';
+
+it('should render the navbar', () => {
+  const component = render(<Router><Navbar /></Router>);
+  expect(component).toMatchSnapshot();
+});
+
+describe('Click events', () => {
+  test('that user to redirected to the Home page on clicking Home', () => {
+    render(<Router><Navbar /></Router>);
+    expect(screen.queryByText(/Welcome/)).toBeNull();
+    userEvent.click(screen.getByText('Home'));
+    expect(screen.queryByText(/Welcome/)).toBeInTheDocument();
+  });
+
+  test('that user to redirected to the Quote page on clicking Quote', () => {
+    render(<Router><Navbar /></Router>);
+    expect(screen.queryByText(/Mathematics is not about numbers/)).toBeNull();
+    userEvent.click(screen.getByText('Quote'));
+    expect(screen.queryByText(/Mathematics is not about numbers/)).toBeInTheDocument();
+  });
+
+  test('that user to redirected to the Calculator page on clicking Calculator', () => {
+    render(<Router><Navbar /></Router>);
+    expect(screen.queryByText('Lets do Some Math:')).toBeNull();
+    userEvent.click(screen.getByText('Calculator'));
+    expect(screen.queryByText('Lets do Some Math:')).toBeInTheDocument();
+  });
+});
```

Comment:
- [ ] Kindly review the navbar test, it's failing... see below

![image](https://user-images.githubusercontent.com/67957533/134473996-f304af81-f95f-49ab-982e-9af0a9b570ac.png)